notes/randomNote/gkeep_api_service.py

```diff
@@ -0,0 +1,15 @@
+import gkeepapi
+import random
+
+
+class gkeepApiService:
```

Comment:
we can remove this code for now as we decided to use another approach of loading notes into database
in case we need this code in future, we'll be able to access it in GitHub:)